MonstieWash/Assets/Scripts/MonsterAi/MonsterBrain.cs

```diff
@@ -28,16 +28,12 @@ protected class MoodData
     #endregion
 
     #region Attacks
-    [Tooltip("Minimum time (inclusive, in seconds) between attack attempts while the monster is aggressive")][SerializeField] private float minBetweenAttacks; // Creates an attack event between the min and max time, if possible.
-    [Tooltip("Maximum time (inclusive, in seconds) between attack attempts while the monster is aggressive")][SerializeField] private float maxBetweenAttacks; // Creates an attack event between the min and max time, if possible.
-    private float m_attackTimer;    // Chosen time to wait before the next attack (randomized between min and max after every attack).
-    private float m_lastAttackTime = 0f;    // Time elapsed since the last attack.
-
     public event Action MonsterAttack;    // Monster attack event.
     #endregion
 
     #region Flinch
     public event Action OnFlinch;
+    private int flinchCount = 0;    // Number of times the monster has flinched in between attacks
```

Comment:
### 🗒️ **Convention** | _This is for maintaining our coding conventions._

Don't forget the lil baby `m_` for private fields.

```suggestion
    private int m_flinchCount = 0;    // Number of times the monster has flinched in between attacks
```